pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -90,7 +97,7 @@ func ExamplePresentation_JWTClaims() {
 `
 
 	// The Holder wants to send the presentation to the Verifier in JWS.
-	vp, err := verifiable.NewPresentation([]byte(vpStrFromWallet))
+	vp, err := verifiable.NewUnverifiedPresentation([]byte(vpStrFromWallet))
```

Comment:
Why this change?

Reply:
`vpStrFromWallet` has no embedded proof. If we pass it to `NewPresentation()` as JSON bytes, it will raise an error saying that no proof was found.
This test did not fail before because of `//nolint:lll` before `//Output:` line which made the test function non-effective. I moved `//nolint:lll` to the line before the method declaration to enable the test. There were about 10 not working example-based tests in `verifiable`.